### PR TITLE
Docs for upgrading the jetstream cluster

### PR DESCRIPTION
**Pull request by scholtzan**
Docs around updating Argo and jetstream

### CODE REVIEW

File: docs/jetstream/operations.md [hunk 1]
```diff
@@ -38,17 +38,42 @@ In addition to running experiment analyses in parallel, [Dask](https://dask.org/
...
+	* Install commonly used components: `kubectl apply -n argo -f https://github.com/argoproj/argo-workflows/releases/download/v3.4.5/install.yaml`
 	* Create new `clusterrole`: `kubectl create rolebinding default-admin --clusterrole=admin --serviceaccount=argo:default --namespace=argo`
 * The [`jetstream` DAG in Airflow](https://github.com/mozilla/telemetry-airflow/blob/master/dags/jetstream.py) triggers the `run-argo` job daily and either requires Compute Engine API access or the parameters `cluster_ip` and `cluster_cert` need to be provided
 	* Currently the Airflow cluster does not have Compute Engine API access, so the cluster IP and certificate are stored as secrets and used for running Jetstream
 
+## Argo Workflow UI
+
+Argo provides a Web UI to access running workflows. Users need to authenticate using a Bearer token:
+* Get Bearer token and copy: `gcloud container clusters get-credentials jetstream --region=us-central1-a && kubectl -n argo exec $(kubectl get pod -n argo -l 'app=argo-server' -o jsonpath='{.items[0].metadata.name}') -- argo auth token`
```
Comment (danielkberry) [inline docs/jetstream/operations.md:55]: I had to add the `CLOUDSDK_CORE_PROJECT=moz-fx-data-experiments` parameter at the beginning to get the command to work (guess I was defaulted to the `moz-fx-data-bq-data-science` project). Maybe we can add this as a note in a sub-bullet point

File: docs/jetstream/operations.md [hunk 2]
```diff
@@ -38,17 +38,43 @@ In addition to running experiment analyses in parallel, [Dask](https://dask.org/
...
 	* Currently the Airflow cluster does not have Compute Engine API access, so the cluster IP and certificate are stored as secrets and used for running Jetstream
 
+## Argo Workflow UI
+
+Argo provides a Web UI to access running workflows. Users need to authenticate using a Bearer token:
+* set `export CLOUDSDK_CORE_PROJECT=moz-fx-data-experiments`
+* Get Bearer token and copy: `gcloud container clusters get-credentials jetstream --region=us-central1-a && kubectl -n argo exec $(kubectl get pod -n argo -l 'app=argo-server' -o jsonpath='{.items[0].metadata.name}') -- argo auth token` 
+* Connect to the Workflow UI using port forwarding: `kubectl -n argo port-forward svc/argo-server 2746:2746`
+* Open [https://localhost:2746](https://localhost:2746)
+* Use the generated Bearer token for authentication
```
Comment (mikewilli) [inline docs/jetstream/operations.md:59]:
```suggestion
* Use the generated Bearer token (including the word `Bearer`) for authentication
```

File: docs/jetstream/operations.md (hunk 1 above)
Comment (mikewilli) [reply docs/jetstream/operations.md:56]: Should we note that warnings about the old auth scheme being deprecated are safe to ignore?

File: docs/jetstream/operations.md [hunk 3]
```diff
@@ -38,17 +38,43 @@ In addition to running experiment analyses in parallel, [Dask](https://dask.org/
...
-	* Install commonly used components: `kubectl apply -n argo -f https://raw.githubusercontent.com/argoproj/argo/stable/manifests/quick-start-postgres.yaml`
+	* Install commonly used components: `kubectl apply -n argo -f https://github.com/argoproj/argo-workflows/releases/download/v3.4.5/install.yaml`
 	* Create new `clusterrole`: `kubectl create rolebinding default-admin --clusterrole=admin --serviceaccount=argo:default --namespace=argo`
 * The [`jetstream` DAG in Airflow](https://github.com/mozilla/telemetry-airflow/blob/master/dags/jetstream.py) triggers the `run-argo` job daily and either requires Compute Engine API access or the parameters `cluster_ip` and `cluster_cert` need to be provided
 	* Currently the Airflow cluster does not have Compute Engine API access, so the cluster IP and certificate are stored as secrets and used for running Jetstream
 
+## Argo Workflow UI
+
+Argo provides a Web UI to access running workflows. Users need to authenticate using a Bearer token:
+* set `export CLOUDSDK_CORE_PROJECT=moz-fx-data-experiments`
```
Comment (mikewilli) [inline docs/jetstream/operations.md:55]: Is this the same as setting the project with `gcloud config set project <project>`?

Comment (scholtzan) [reply docs/jetstream/operations.md:55]: I think it is not _quite_ the same. Last time Daniel tried setting the project like this and it still didn't work...

**Pull request merged**